src/CMakeLists.txt

```diff
@@ -1,14 +1,30 @@
+file(MAKE_DIRECTORY "${SM_SRC_DIR}/generated")
+
+if(NOT ANDROID)
 # Version Updater library.
 add_subdirectory(version_updater)
 
 # This part may move to a different file: it's the SM Data project.
 add_custom_command(
-  OUTPUT ${SM_SRC_DIR}/verstub.cpp
-  COMMAND VersionUpdater ${SM_SRC_DIR}/verstub.cpp
+  OUTPUT ${SM_SRC_DIR}/generated/verstub.cpp
+  COMMAND VersionUpdater ${SM_SRC_DIR}/generated/verstub.cpp
```

Comment:
I thought part of the idea was that there would not be a version updater program.